lib/helpers/stake/gateway_composer/Facilitator.js

```diff
@@ -54,52 +43,47 @@ class Facilitator {
    *
    * Note: Add KYC worker account/private key in web3 wallet before calling acceptStakeRequest.
    *
-   * @param stakeRequestHash Stake request hash unique for each stake.
-   * @param signature Signature object format:
-   *                  {
-   *                    messageHash: signHash,
-   *                    v: vrs[0],
-   *                    r: vrs[1],
-   *                    s: vrs[2],
-   *                    signature: signature
-   *                  }
-   * @param bountyInWei Bounty amount in wei's that needs to be approved.
-   * @param valueTokenAbi ValueToken contract ABI.
-   * @param hashLock HashLock of facilitator.
-   * @param originWeb3 Origin chain web3 object.
-   * @param txOptions - Tx options.
+   * @param {string} stakeRequestHash Stake request hash unique for each stake.
+   * @param {Object} signature Signature object format:
+   * @param {string} signature.messageHash The hash of the signed message.
+   * @param {string} signature.v V of the signature.
+   * @param {string} signature.r R of the signature.
+   * @param {string} signature.s S of the signature.
+   * @param {string} signature.signature Signature.
+   * @param {string} bountyInWei Bounty amount in wei's that needs to be approved.
+   * @param {string} hashLock Hashed secret of facilitator.
+   * @param {Object} txOptions Transaction options used with web3.
    */
   async acceptStakeRequest(
     stakeRequestHash,
     signature,
     bountyInWei,
-    valueTokenAbi,
     hashLock,
-    originWeb3,
     txOptions,
   ) {
-    const oThis = this;
-
     const stakeHelperInstance = new StakeHelper(
-      oThis.originWeb3,
-      oThis.brandedToken,
-      oThis.gatewayComposer,
+      this.originWeb3,
```

Comment:
I think `stakeHelperInstance` should also be injected as a dependency from the constructor. Optionally, can be instantiated in the constructor.  Unit testing this part will be tough due to object creation is done here.